docs/get-started/get-started.md

```diff
@@ -239,4 +242,172 @@ CompositionLocalProvider(LocalGravatarTheme provides object : GravatarTheme {
 }) {
     LargeProfileSummary(profile = userProfile)
 }
+```
+
+## Quick Editor
+
+The `:gravatar-quickeditor` module provides a fully featured component that allows the user to modify their avatar without leaving your app. 
+
+To do that the QuickEditor needs an authorization token to perform requests on behalf of the user. There are two ways for that:
+
+### 1. Obtain the token yourself and provide it to the Quick Editor
+
+Quick Editor can be launched with the provided token. To obtain it, you have to follow the [OAuth docs](https://docs.gravatar.com/?page_id=2555&preview=true) and implement the OAuth flow within your app.
+
+Once you have the token, here's how you can embed the QuickEditor in your Compose screen:
+
+```kotlin
+var showBottomSheet by rememberSaveable { mutableStateOf(false) }
+
+if (showBottomSheet) {
+    GravatarQuickEditorBottomSheet(
+        gravatarQuickEditorParams = GravatarQuickEditorParams {
+            email = Email("{USER_EMAIL}")
+            avatarPickerContentLayout = AvatarPickerContentLayout.Horizontal
+        },
+        authenticationMethod = AuthenticationMethod.Bearer("{TOKEN}"),
+        onAvatarSelected = { avatarUpdateResult -> ... },
+        onDismiss = { gravatarQuickEditorDismissReason ->
+            showBottomSheet = false
+            ...
+        },
+    )
+}
+```
+
+### 2. Leave the OAuth flow to Quick Editor
+
+Quick Editor can handle the heavy lifting of running the full OAuth flow, so you don't have to do that. We will still need a few things from you.
+First, you have to go to [OAuth docs](https://docs.gravatar.com/?page_id=2555&preview=true) and create your Application. Define the `Redirect URLs`.
+
+> Keep in mind that you need to use the `https` scheme. Internally, QuickEditor uses Implicit OAuth flow (`response_type=token`) and for security reasons, the server doesn't allow custom URL schemes.
+
+For the sake of this example let's assume the redirect URL is `https://gravatar.com/redirect-url`.
+
+In your `AndroidManifest.xml` you need to add an `<intent-filter>` and the `android:launchMode="singleTask"` to the activity that will 
+launch the Quick Editor (or the last/main activity depending on your app architecture). This is important because the Quick Editor will be waiting for the `onNewIntent()` callback to handle OAuth redirection.
+
+```xml
+<activity
+    android:name=".YourActivity"
+    android:launchMode="singleTask"
+    ...>
+    <intent-filter android:autoVerify="true">
+        <action android:name="android.intent.action.VIEW" />
+
+        <category android:name="android.intent.category.DEFAULT" />
+        <category android:name="android.intent.category.BROWSABLE" />
+
+        <data
+            android:scheme="https"
+            android:host="gravatar.com"
+            android:pathPrefix="/redirect-url"
```

Comment:
Is it confusing if I use the real example here? Should I use sth like: {SCHEME} instead?

Reply:
As we required it to be `https` I would keep the scheme as it is but just modify the host:

```suggestion
        <data
            android:scheme="https"
            android:host="{YOUR-HOST}"
            android:pathPrefix="/redirect-url"
```

Or maybe we can use another example URL instead of the one containing "gravatar.com" because, yeah, it can be confusing. Something like: `https://yourhost.com/redirect-url`

Reply:
Done